src/components/puzzles/NewCss.tsx

```diff
@@ -115,6 +138,11 @@ const NewCss: FunctionalComponent<NewCssPropsType> = ({ state, onChange }) => {
           <input class={state?.solution?.valid ? style.inputValid : style.inputInvalid} value={state?.solution?.value || ''} onInput={onSolutionChange} />
           {state?.solution?.message && <div class={style.errorMessage}>{state?.solution?.message}</div>}
         </div>
+        <div class={style.solutionLengthLimit}>
+          <div class={style.propName}>Solution length limit (amount of characters):</div>
+          <input type="number" min={-1} value={state?.solutionLengthLimit?.value} onInput={onSolutionLengthLimitChange} />
```

Comment:
What does the `-1` solution length limit mean?

Reply:
No limit. Using a number here just to avoid UX complexity: ideally it should be a checkbox + numeric input.